_pages/business-units/18f/projects-partners/leading-projects.md

```diff
@@ -54,7 +55,8 @@ Some things they’ll help you navigate, find, and use:
 * If you need documentation, look to the [handbook]({{site.baseurl}}) and our [guides](https://18f.gsa.gov/guides/).
 * If you need support in a specific skill set or help with deliverables, get help by using a [microrequest](https://gsa-tts.slack.com/messages/microrequests/).
 * If your project is part of a portfolio, the portfolio director and your colleagues in the portfolio can be a great resource.
-* If you need advice or coaching, look to the chapters ([#product](https://gsa-tts.slack.com/messages/product/), [#design](https://gsa-tts.slack.com/messages/design/), [#acquisition](https://gsa-tts.slack.com/messages/acquisition/), and [#dev](https://gsa-tts.slack.com/messages/dev/) and guilds ([#g-accessibility](https://gsa-tts.slack.com/messages/g-accessibility/), [#g-research](https://gsa-tts.slack.com/messages/g-research/), [#g-content](https://gsa-tts.slack.com/messages/g-content/), [#g-diversity](https://gsa-tts.slack.com/messages/g-diversity/), [#g-devops](https://gsa-tts.slack.com/messages/g-devops/).
+* If you need advice or coaching, look to the chapters ([#product](https://gsa-tts.slack.com/messages/product/), [#design](https://gsa-tts.slack.com/messages/design/), [#acquisition](https://gsa-tts.slack.com/messages/acquisition/), and [#dev](https://gsa-tts.slack.com/messages/dev/) as well as our [guilds and working groups]({{site.baseurl}}/working-groups-and-guilds-101/). Each of them have a corresponding Slack channel you can jump into any time.
```

Comment:
```suggestion
* If you need advice or coaching, look to the [chapters]({{site.baseurl}}/#18f-chapters) as well as our [guilds and working groups]({{site.baseurl}}/working-groups-and-guilds-101/). Each of them have a corresponding Slack channel you can jump into any time.
```

Reply:
I thought about it, but given that page doesn't appear to be linked from the index and I'm not sure how fresh it is, figured I'd leave it to the list of slack channels. They're not wrong. I could go either way, but I don't have a high level of confidence in that page.